npc/re/quests/juno_monster_society.txt

```diff
@@ -18,9 +18,9 @@ rachel,142,155,3	script	Carbonated Water Vending Machine#ra	4_VENDING_MACHINE2,{
 	next;
 	mes "[Carbonated Water Vending Machine]";
 	mes "<font size = 10, color = DF0101>Caution: There is a report that excessive consumption of the product in a state of accumulated fatigue due to its strong carbonation effect may cause temporary memory loss.</font>";
-	next;
+	close2;
 	callshop "market_soda";
-	close;
+	end;
```

Comment:
Please keep commented the old (official) behaviour. We just need to adapt our scripting engine to keep the original behavior